crates/provider-sdk/src/lib.rs

```diff
@@ -142,3 +150,107 @@ pub trait ProviderHandler: Sync {
     /// Handle system shutdown message
     async fn shutdown(&self) {}
 }
+
+/// Parse a `wit_parser::Resolve` from anything that implements a mechanism for looking up file content
+async fn parse_wit_from_file_content_lookup(
+    _lookup: Box<dyn FileLookup + Send>,
+) -> ProviderInitResult<Resolve> {
+    // TODO: use a file lookup to build a Resolve
+    Ok(Resolve::default())
+}
+
+/// Generate a list of incoming NATS RPC topics on which the provider should listen
+fn generate_incoming_rpcs_topics(
+    lattice_name: impl AsRef<str>,
+    wrpc_version: impl AsRef<str>,
+    resolve: &Resolve,
+) -> ProviderInitResult<Vec<String>> {
+    let lattice_name = lattice_name.as_ref();
+    let wrpc_version = wrpc_version.as_ref().replace(".", "!");
```

Comment:
We're going to start with just using `.` everywhere, no need for this replacement

Reply:
Reverted!